libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -99,7 +99,7 @@
 // MegaSquirt EFI protocol
 #define AP_SERIALMANAGER_EFI_MS_BAUD           115
 #define AP_SERIALMANAGER_EFI_MS_BUFSIZE_RX     512
-#define AP_SERIALMANAGER_EFI_MS_BUFSIZE_TX     16
+#define AP_SERIALMANAGER_EFI_MS_BUFSIZE_TX     32
```

Comment:
This doesn't look right.  If you need the larger buffer you can do your own `begin()` within the Hirth backend.

Reply:
We have reverted this back to original.